server/mongodb/models/Chapter.ts

```diff
@@ -1,13 +1,15 @@
-import mongoose, { Schema } from "mongoose";
+import mongoose, { CallbackError, Schema, Types } from "mongoose";
 import AddressSchema from "server/mongodb/models/embedded/Address";
+import ProjectModel from "server/mongodb/models/Project";
+import UserModel from "server/mongodb/models/User";
+import { deleteDocumentAndDependencies } from "server/utils/delete-document";
 import { Chapter, MaintenanceType } from "src/utils/types";
 
 const ChapterSchema = new Schema<Chapter>({
   name: {
     type: String,
     required: true,
   },
-  email: String,
```

Comment:
This probably should have been deleted when the `contact` field was added

Reply:
Yeah I think I agree with that.  Thoughts @shashanklal01?

Reply:
Sounds good to me